### ISSUE TITLE
Linux GPU Rendering Screen Flicker on New Version of LibGL

### ISSUE THREAD

**Issue by ghost** [runelite/runelite#13024]
**Describe the bug**
When walking around with GPU rendering enabled the screen flickers blank for an instant whenever a new chunk is loaded. The issue began last night, I'm not sure but I think it's related to the newest version of LibGL released on Linux Mint (1.3.2-1)

Disabling fog eliminated the screen flicker.

**To Reproduce**
1. Launch RL on Linux with GPU rendering and fog enabled and LibGL version 1.3.2-1 or later
2. Run far enough to load a new chunk

**Environment (please complete the following information):**
 - OS: Linux Mint 20
 - RuneLite version: 1.6.36.1
 - Launcher version: 2.1.3

**Comment by Nightfirecat** [comment]
Could you upload your client logs from today, or some session where this bug occurred?

**Comment by Nightfirecat** [comment]
Also this same issue reportedly occurred for another user on a Macbook Air

**Comment by jdl2803** [comment]
```
2021-01-07 00:14:07 [pool-5-thread-1] WARN  n.r.c.plugins.worldhopper.ping.Ping - error pinging
java.net.UnknownHostException: oldschool48.runescape.com: Name or service not known
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(Unknown Source)
	at java.base/java.net.InetAddress.getAddressesFromNameService(Unknown Source)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName0(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at java.base/java.net.InetAddress.getByName(Unknown Source)
	at net.runelite.client.plugins.worldhopper.ping.Ping.tcpPing(Ping.java:99)
	at net.runelite.client.plugins.worldhopper.ping.Ping.ping(Ping.java:55)
	at net.runelite.client.plugins.worldhopper.WorldHopperPlugin.ping(WorldHopperPlugin.java:848)
	at net.runelite.client.plugins.worldhopper.WorldHopperPlugin.pingNextWorld(WorldHopperPlugin.java:806)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
2021-01-07 01:37:09 [Client] WARN  n.r.c.p.c.clues.HotColdClue - Hot cold solver could not be initialized, clue type is unknown; text: Buried beneath the ground, who knows where it's found. Lucky for you, A man called Reldo may have a clue., npc: Reldo, solution: Speak to Reldo to receive a strange device.
2021-01-07 01:37:09 [Client] WARN  n.r.c.p.c.clues.HotColdClue - Hot cold solver could not be initialized, clue type is unknown; text: Buried beneath the ground, who knows where it's found. Lucky for you, A man called Jorral may have a clue., npc: Jorral, solution: Speak to Jorral to receive a strange device.
2021-01-07 01:37:09 [Client] WARN  n.r.c.p.c.clues.HotColdClue - Hot cold solver could not be initialized, clue type is unknown; text: Buried beneath the ground, who knows where it's found. Lucky for you, A man called Watson may have a clue., npc: Watson, solution: Speak to Watson to receive a strange device.
2021-01-07 02:40:28 [OkHttp https://services.runescape.com/...] WARN  n.r.http.api.hiscore.HiscoreClient - Jagex Hiscore API returned unexpected data
2021-01-07 14:56:31 [pool-5-thread-1] WARN  n.r.c.plugins.worldhopper.ping.Ping - error pinging
java.net.UnknownHostException: oldschool232.runescape.com: Name or service not known
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(Unknown Source)
	at java.base/java.net.InetAddress.getAddressesFromNameService(Unknown Source)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName0(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at java.base/java.net.InetAddress.getByName(Unknown Source)
	at net.runelite.client.plugins.worldhopper.ping.Ping.tcpPing(Ping.java:99)
	at net.runelite.client.plugins.worldhopper.ping.Ping.ping(Ping.java:55)
	at net.runelite.client.plugins.worldhopper.WorldHopperPlugin.ping(WorldHopperPlugin.java:848)
	at net.runelite.client.plugins.worldhopper.WorldHopperPlugin.pingNextWorld(WorldHopperPlugin.java:806)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
2021-01-07 17:38:01 [OkHttp https://pbs.twimg.com/...] WARN  n.r.client.plugins.feed.FeedPanel - null
java.net.UnknownHostException: pbs.twimg.com: Name or service not known
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(Unknown Source)
	at java.base/java.net.InetAddress.getAddressesFromNameService(Unknown Source)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName0(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at okhttp3.Dns$1.lookup(Dns.java:39)
	at okhttp3.internal.connection.RouteSelector.resetNextInetSocketAddress(RouteSelector.java:170)
	at okhttp3.internal.connection.RouteSelector.nextProxy(RouteSelector.java:136)
	at okhttp3.internal.connection.RouteSelector.next(RouteSelector.java:81)
	at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:171)
	at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:121)
	at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:100)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:120)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:67)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:185)
	at okhttp3.RealCall$AsyncCall.execute(RealCall.java:135)
	at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
2021-01-07 17:38:11 [pool-5-thread-1] WARN  n.r.c.plugins.worldhopper.ping.Ping - error pinging
java.net.UnknownHostException: oldschool78a.runescape.com: Name or service not known
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(Unknown Source)
	at java.base/java.net.InetAddress.getAddressesFromNameService(Unknown Source)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName0(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at java.base/java.net.InetAddress.getByName(Unknown Source)
	at net.runelite.client.plugins.worldhopper.ping.Ping.tcpPing(Ping.java:99)
	at net.runelite.client.plugins.worldhopper.ping.Ping.ping(Ping.java:55)
	at net.runelite.client.plugins.worldhopper.WorldHopperPlugin.ping(WorldHopperPlugin.java:848)
	at net.runelite.client.plugins.worldhopper.WorldHopperPlugin.pingNextWorld(WorldHopperPlugin.java:806)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
2021-01-07 18:19:36 [AWT-EventQueue-0] WARN  n.r.c.plugins.hiscore.HiscorePanel - Error fetching Hiscore data timeout
2021-01-07 19:25:19 [pool-5-thread-1] WARN  n.r.c.plugins.worldhopper.ping.Ping - error pinging
java.net.UnknownHostException: oldschool195.runescape.com: Name or service not known
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(Unknown Source)
	at java.base/java.net.InetAddress.getAddressesFromNameService(Unknown Source)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName0(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at java.base/java.net.InetAddress.getByName(Unknown Source)
	at net.runelite.client.plugins.worldhopper.ping.Ping.tcpPing(Ping.java:99)
	at net.runelite.client.plugins.worldhopper.ping.Ping.ping(Ping.java:55)
	at net.runelite.client.plugins.worldhopper.WorldHopperPlugin.ping(WorldHopperPlugin.java:848)
	at net.runelite.client.plugins.worldhopper.WorldHopperPlugin.pingNextWorld(WorldHopperPlugin.java:806)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
2021-01-07 19:51:25 [AWT-EventQueue-0] WARN  n.r.c.plugins.hiscore.HiscorePanel - Error fetching Hiscore data timeout
2021-01-07 21:39:32 [pool-5-thread-1] WARN  n.r.c.plugins.worldhopper.ping.Ping - error pinging
java.net.UnknownHostException: oldschool12.runescape.com: Name or service not known
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(Unknown Source)
	at java.base/java.net.InetAddress.getAddressesFromNameService(Unknown Source)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName0(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
	at java.base/java.net.InetAddress.getByName(Unknown Source)
	at net.runelite.client.plugins.worldhopper.ping.Ping.tcpPing(Ping.java:99)
	at net.runelite.client.plugins.worldhopper.ping.Ping.ping(Ping.java:55)
	at net.runelite.client.plugins.worldhopper.WorldHopperPlugin.ping(WorldHopperPlugin.java:848)
	at net.runelite.client.plugins.worldhopper.WorldHopperPlugin.pingNextWorld(WorldHopperPlugin.java:806)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.runAndReset(Unknown Source)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
2021-01-07 21:56:23 [OkHttp https://services.runescape.com/...] WARN  n.r.http.api.hiscore.HiscoreClient - Jagex Hiscore API returned unexpected data
2021-01-07 22:54:43 [pool-3-thread-1] INFO  n.r.client.discord.DiscordService - Discord RPC service is ready with user bpf101.
2021-01-07 23:11:27 [RuneLite] INFO  net.runelite.client.RuneLite - RuneLite 1.6.36.1 (launcher version 2.1.3) starting up, args: none
2021-01-07 23:11:28 [Preloader] INFO  net.runelite.client.rs.ClientLoader - Using cached vanilla client
2021-01-07 23:11:29 [Preloader] WARN  net.runelite.client.rs.ClientLoader - Cached patch hash mismatches, regenerating patch
2021-01-07 23:11:31 [Preloader] INFO  net.runelite.client.rs.ClientLoader - client-patch 948284500.7
2021-01-07 23:11:31 [RuneLite] INFO  n.r.client.account.SessionManager - No session file exists
2021-01-07 23:11:32 [RuneLite] INFO  n.r.c.plugins.kourendlibrary.Library - Library is now reset
2021-01-07 23:11:33 [RuneLite] INFO  n.r.c.e.ExternalPluginManager - Loading external plugin "equipment-inspector" version "1.1" commit "09fed5837d1d60e173704a9ff110d72577306ba1"
2021-01-07 23:11:33 [RuneLite] INFO  n.r.c.e.ExternalPluginManager - Loading external plugin "zoom" version "1.0" commit "a83bf325068ffdc25b3726d982f5fd5f358c5d7a"
2021-01-07 23:11:33 [RuneLite] INFO  n.r.c.e.ExternalPluginManager - Loading external plugin "door-kicker" version "1.0" commit "04dfc0e0974b28080599bb1ffdc3f7195df922d6"
2021-01-07 23:11:33 [RuneLite] INFO  n.r.c.e.ExternalPluginManager - Loading external plugin "resource-packs" version "1.3.1" commit "003799ef17617b6b419e8e49808f593269595ed4"
2021-01-07 23:11:33 [RuneLite] INFO  n.r.c.e.ExternalPluginManager - Loading external plugin "fight-cave-waves" version "1.1" commit "9ccbfe656e472a2bdd897a7a7c27d0caf23ec867"
2021-01-07 23:11:33 [RuneLite] INFO  n.r.c.e.ExternalPluginManager - Loading external plugin "menu-swapper-extended" version "1.6" commit "4a2c8d1edd7f490e0c92c5767ee36eda74b5804c"
2021-01-07 23:11:33 [RuneLite] INFO  n.r.c.e.ExternalPluginManager - Loading external plugin "calculator" version "1.1.0" commit "9a3f0714c9e0169f9a9cbefcce67c3742a893f30"
2021-01-07 23:11:33 [RuneLite] INFO  n.r.c.e.ExternalPluginManager - Loading external plugin "quest-helper" version "1.2.2" commit "93c4a7e376c8bb47863bfe67fdd879b5f42396e5"
2021-01-07 23:11:33 [RuneLite] INFO  n.r.c.e.ExternalPluginManager - Loading external plugin "banked-experience" version "1.6.3" commit "6c7e389519faef31f086d34cb07c75a0c31b1ed9"
2021-01-07 23:11:34 [RuneLite] INFO  n.r.client.discord.DiscordService - Initializing Discord RPC service.
2021-01-07 23:11:36 [pool-3-thread-1] INFO  n.r.client.discord.DiscordService - Discord RPC service is ready with user bpf101.
2021-01-07 23:11:37 [AWT-EventQueue-0] INFO  c.t.calculator.CalculatorPlugin - calculator started!
2021-01-07 23:11:38 [RuneLite] INFO  net.runelite.client.RuneLite - Client initialization took 11105ms. Uptime: 18138ms
2021-01-07 23:11:39 [AWT-EventQueue-0] INFO  net.runelite.client.ui.ClientUI - Showing frame net.runelite.client.ui.ContainableFrame[frame0,0,28,1678x1009,layout=java.awt.BorderLayout,title=RuneLite,resizable,normal,defaultCloseOperation=DO_NOTHING_ON_CLOSE,rootPane=javax.swing.JRootPane[,0,0,1678x1009,layout=org.pushingpixels.substance.internal.ui.SubstanceRootPaneUI$SubstanceRootLayout,alignmentX=0.0,alignmentY=0.0,border=org.pushingpixels.substance.internal.utils.border.SubstancePaneBorder@7d119798,flags=16777673,maximumSize=,minimumSize=,preferredSize=],rootPaneCheckingEnabled=true]
```

**Comment by jdl2803** [comment]
I restarted the client at 11:11 and the flickering continued until I disabled fog a few minutes later.

**Comment by kmarius** [comment]
I can reproduce this on arch linux, however with all 3 most recent libgl versions (1.3.0 to 1.3.2). Can you get a log of the recently updated packages? 

I recorded the flicker and stepped through the frames, this is what it looks like. The eleven or so frames before the chunk is loaded are still and the first frame with the new chunk contains the flicker.

![2021-01-10_01:31:56-122505125](https://user-images.githubusercontent.com/5224719/104111657-e4d33480-52e4-11eb-94f8-7c031df3fa08.png)
![2021-01-10_01:31:58-706589235](https://user-images.githubusercontent.com/5224719/104111663-f3215080-52e4-11eb-8aea-9ad36d16131c.png)
![2021-01-10_01:32:01-651345643](https://user-images.githubusercontent.com/5224719/104111664-f4eb1400-52e4-11eb-9d90-b684a68b91ad.png)

**Comment by Adam-** [comment]
OP has deleted their github account, so we can probably just close this?

**Comment by Adam-** [comment]
Nvm - this seems like a valid issue & I can reproduce it on Windows.

**Comment by Adam-** [comment]
fixed in 359b909d8e29ebe2b2589aefd74b7159715b805e
